lib/misc.py

```diff
@@ -33,6 +34,13 @@ def __call__(self, *args, **kwargs):
 # Argument: command line arguments Namespace. Do not need to call sys.exit()
 # because caller manages that.
 
+def delete(cli):
+   imgdir = cli.storage + '/img/' + cli.image_ref
```

Comment:
Do we have a way to, given an image reference, find the directory it's unpacked in? If so, use it here; if not, consider creating one.

Reply:
On second thought, maybe better to:

1. If one's available, use it.
2. If not, file an issue to create one. Perhaps a class in `charliecloud.py`? This could benefit Jordan's work on architecture subdirectories too.